web/gds-user-ui/src/components/Section/SearchDirectory.tsx

```diff
@@ -121,11 +121,13 @@ const SearchDirectory: React.FC<TSearchDirectory> = ({
                     </AutoComplete>
 
                     <Button
-                      isLoading={isLoading}
                       variant="outline"
-                      type="submit"
+                      onClick={(e: FormEvent) => {
+                        onResetData();
+                        formRef.current?.removeItem(search);
+                      }}
                       spinnerPlacement="start">
-                      <SearchIcon />
+                      Clear
```

Comment:
This is a nice improvement since users no longer needed to click the search icon to see their results.